app/src/common/shared/com/igalia/wolvic/VRBrowserActivity.java

```diff
@@ -2018,7 +2019,10 @@ public boolean isPassthroughSupported() {
         return DeviceType.isOculusBuild() || DeviceType.isLynx() || DeviceType.isSnapdragonSpaces() ||
                (DeviceType.isPicoXR() && Build.ID.compareTo(kPicoVersionPassthroughUpdate) >= 0);
     }
-
+    @Override
+    public boolean isHandTrackingOnlyDevice() {
+        return DeviceType.isLynx() || DeviceType.isSnapdragonSpaces() || DeviceType.isMagicLeap2();
```

Comment:
> You should instead invoke a method from native informing the Java side that hand tracking is supported. We do that already for things like eye tracking.

But we are not interested on knowing whether handtracking is supported or not, but whether there is no controllers.

Reply:
Now that you mention, we probably don't want to offer this feature on devices that don't support hand tracking at all, but that's less concerning. What we want to avoid with this is offering the feature on devices where hand tracking is the only way to control the application.